modules/core/src/main/java/org/apache/ignite/spi/deployment/local/LocalDeploymentSpi.java

```diff
@@ -146,6 +154,68 @@
         return null;
     }
 
+    /** {@inheritDoc} */
+    @Nullable @Override public DeploymentResource findResource(String rsrcName, @Nullable ClassLoader clsLdr) {
```

Comment:
Please say to me honest: does this method can look for remote resource?

Reply:
GridLocalDeploymentSpiSelfTest#testDeploy seems asks your question ?